views/result.erb

```diff
@@ -0,0 +1,9 @@
+<div class="centered-content">
+  <form action="/"> 
+    <h1><%=@player.name%> vs. Computer</h1>
+    <h2 id="results-player"><%=@player.name%> chose <%=@player.choice%></h2>
+    <h2 id="results-computer">Computer chose <%=@computer_choice%></h2>
+    <h2 id="results-winner"><%=@winner%></h2>
+    <input for="play-again" name="play-again" class="button" id="play-again-button" type="submit" value="Play Again">
+  <form>
+</div>
```

Comment:
Brilliant program, shows clear understanding of the dry principle and single responsibility principle throughout. Very well done!